platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/notifications/commands/SendCommand.java

```diff
@@ -113,7 +113,7 @@ public Object execute() throws Exception {
         try {
           Thread.sleep(waitTime * 1000L);
         } catch (InterruptedException e) {
-
+          Thread.currentThread().interrupt();
```

Comment:
❗️ Need to compute end time outside of a while loop for this instance of i and compute the remaining time inside. Interruption should loop back for this particular loop. Outside the overall for(), propagate the interruption in a finally clause